internal/lib/lib.go

```diff
@@ -1079,6 +1079,10 @@ func GetClusterIDSplit() string {
 	if clusterID != "" {
 		clusterName := strings.Split(clusterID, ":")
 		if len(clusterName) > 1 {
+			if GetVPCMode() {
+				// Include first 5 characters to add more uniqueness to cluster name
+				return clusterName[0] + "-" + clusterName[1][:5]
```

Comment:
Will this logic not impact VPC based deployments on upgrade ?

Reply:
No, VPC based deployment will start from VC 9.0 where this logic is aimed to go